### PR TITLE
Feature/inital implementation

### PR DESCRIPTION
**Pull request by zfhui**

### CODE REVIEW

File: silent_ping.gemspec [hunk 1]
```diff
@@ -9,9 +9,9 @@ Gem::Specification.new do |spec|
   spec.authors       = ["Zhuo-Fei Hui"]
   spec.email         = ["fei@blinkist.com"]
 
-  spec.summary       = %q{TODO: Write a short summary, because RubyGems requires one.}
-  spec.description   = %q{TODO: Write a longer description or delete this line.}
-  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
+  spec.summary       = %q{ Provides an /ping endpoint for Rack health check, that doesn't get's logged. }
```
Comment (zfhui) [inline silent_ping.gemspec]:
```suggestion
  spec.summary       = %q{ Provides an /ping endpoint for Rack health check, that doesn't get logged. }
```

File: silent_ping.gemspec [hunk 2]
```diff
@@ -9,9 +9,9 @@ Gem::Specification.new do |spec|
...
+  spec.description   = %q{ Provides an /ping endpoint for Rack health check, that doesn't get's logged. }
```
Comment (zfhui) [inline silent_ping.gemspec]:
```suggestion
  spec.description   = %q{ Provides an /ping endpoint for Rack health check, that doesn't get logged. }
```

**Review by hosseintoussi — APPROVED** [review]
yey - bump your version and 🚢

**Comment by hosseintoussi** [comment]
just one thing! can you benchmark this against our old ping gem?

**Comment by zfhui** [comment]
> just one thing! can you benchmark this against our old ping gem?

I was considering this, but it's doing nothing more than the old gem, it's actually more or less the same code. So what are we benchmarking? 😅

**Comment by zfhui** [comment]
> yey - bump your version and 🚢

This is the 0.1.0 version. The code in master is just `bundle new gem_name`. So, I'll merge this and create the tag afterwards.

**Comment by hosseintoussi** [comment]
>  So what are we benchmarking? 

We need to benchmark that speed wise they are the same. if you think they are the same, then great, benchmarks would validate that. If this is slower, then we would know the price.

**Pull request merged**